### ISSUE TITLE
Close tooltips upon closing keyboard

### ISSUE THREAD

**Issue by Glitchy-Tozier** [florisboard/florisboard#2414]
### Short description

This is really nitpicky, but  tooltips get messed up when closing the keyboard. Sometimes they flicker upon reopening the keyboard, sometimes they can't be opened again for a few minutes, etc.
To fix this, they should probably be closed upon closing the keyboard.

https://github.com/florisboard/florisboard/assets/59611881/33096bd5-5dce-4a24-8f37-1e279ab95685

### Steps to reproduce

1. Open keyboard
2. Long click symbol in smart-bar to trigger tooltip
3. Quickly close the keyboard
4. Quickly reopen the keyboard
5. See bug

### FlorisBoard Version

0.4.0-beta01 (93)

### Install Source

Google PlayStore

### Device

Samsung Galaxy S9

### Android

10

**Comment by lm41** [comment]
I cannot reproduce this behavior at the moment. Is it still present in the latest beta?

**Comment by Glitchy-Tozier** [comment]
I can still reproduce it in `0.4.0-beta02 (94)`.
It's a quick flicker, so depending on your background color, it can be difficult to spot.

**Comment by alvarofelipe12** [comment]
Checked on 0.4.6
It is not happening. @Glitchy-Tozier could you confirm?

https://github.com/user-attachments/assets/87e014e1-c9d3-4e9f-a71a-1fed66292845

**Comment by Glitchy-Tozier** [comment]
Still happens with `0.4.6 (104)`.

https://github.com/user-attachments/assets/0f03974d-9a29-4737-bb2e-d6afb26b12ba

**Comment by lm41** [comment]
Is this occurring in v0.5.0-alpha04 or newer? We've updated the tool-tip dependency with that release.

**Comment by Glitchy-Tozier** [comment]
Yep, it still occurs in `0.5.0-beta02 (110)`. You have to be quick.
